drivers/lxc/lxc.go

```diff
@@ -106,23 +125,58 @@ func (d *Driver) mountVolumes(c *lxc.Container, cfg *drivers.TaskConfig, taskCon
 
 		if filepath.IsAbs(paths[0]) {
 			if !volumesEnabled {
-				return fmt.Errorf("absolute bind-mount volume in config but volumes are disabled")
+				return nil, fmt.Errorf("absolute bind-mount volume in config but volumes are disabled")
 			}
 		} else {
 			// Relative source paths are treated as relative to alloc dir
 			paths[0] = filepath.Join(cfg.TaskDir().Dir, paths[0])
 		}
 
-		mounts = append(mounts, fmt.Sprintf("%s %s none rw,bind,create=dir", paths[0], paths[1]))
+		// LXC assumes paths are relative with respect to rootfs
+		target := strings.TrimLeft(paths[1], "/")
```

Comment:
This is potentially behavior changing - but currently we require customers to insert container paths as "relative" paths, e.g. `"/path/on/host:path/in/container"` [1].  If a uses an absolute path, the bind mount would fail silently without emitting any error.

Thus we strip the prefix `/`, if present, to be nicer to users.

[1] https://www.nomadproject.io/docs/drivers/lxc.html#volumes